package.json

```diff
@@ -53,6 +53,8 @@
     "webpack-utf8-bom": "^1.3.0"
   },
   "dependencies": {
+    "@emotion/react": "^11.10.4",
+    "@emotion/styled": "^11.10.4",
```

Comment:
I believe you can undo these changes here, and also on `package-lock.json`.

Reply:
@murilovmachado It's done.